apps/meteor/client/views/room/MessageList/components/MessageContent.tsx

```diff
@@ -45,7 +45,12 @@ const MessageContent: FC<{ message: IMessage; sequential: boolean; subscription?
 			<MessageBody data-qa-type='message-body'>
 				<MessageRender message={message} />
 			</MessageBody>
-			{message.blocks && <MessageBlock mid={message._id} blocks={message.blocks} appId rid={message.rid} />}
+
+			{message.blocks && (
+				<Box maxWidth='400px'>
+					<MessageBlock mid={message._id} blocks={message.blocks} appId rid={message.rid} />
+				</Box>
+			)}
```

Comment:
create a proper style inside fuselage for message block for that case, pay attention other blocks should stay as  100% width